rust_icu_common/src/lib.rs

```diff
@@ -333,3 +336,24 @@ mod tests {
         let _c_array = CStringVec::new(&values).expect_err("should fail");
     }
 }
+
+/// A zero-value parse error, used to initialize types that get passed into FFI code.
+pub static NO_PARSE_ERROR: sys::UParseError = sys::UParseError {
+    line: 0,
+    offset: 0,
+    preContext: [0; 16usize],
+    postContext: [0; 16usize],
+};
+
+/// Converts a parse error to a Result.
+pub fn parse_ok(e: sys::UParseError) -> Result<(), crate::Error> {
+    if e == NO_PARSE_ERROR {
+        return Ok(());
+    }
+    Err(Error::Wrapper(anyhow!(
+        "parse error: line: {}, offset: {}",
+        e.line,
+        e.offset
+    )))
```

Comment:
Would there be any value in printing `preContext` and `postContext`?

Reply:
IIRC the code that we have wrapped so far does not fill out those two values.